ss_wave_2.rb

```diff
@@ -0,0 +1,219 @@
+
+class SolarSystem
+  attr_accessor :ss
+
+  def initialize(planet_1, planet_2 )
+    @ss = [planet_1, planet_2]
+
```

Comment:
While this certainly works, we were expecting you to pass a list of planets directly into `SolarSystem.new`, rather than passing planets one by one. So something like:

```ruby
class SolarSystem
  attr_accessor :ss
  def initialize(planets)
    @ss = planets
  end
  # ... rest of the class ...
end

planet_a = Planet.new("Jupiter","1882","details")
planet_b = Planet.new("Uranus", "1932","about")
planet_list = [planet_a, planet_b]

my_ss = SolarSystem.new(planet_list)
```

That way whoever creates the instance of `SolarSystem` can create it with as many planets as needed.